### PR TITLE
updated Control Tester

### PR DESCRIPTION
**Pull request by shaiTheKimhi**

### CODE REVIEW

File: Modules/Control/Control.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 __radius = 7370 * 1000
-__circle_time = 90 * 60
+__circle_time = 1 * 60
```
Comment (ron-popov) [inline Modules/Control/Control.py:8]: How about setting these values as constants in a config file ?

File: Tests/Control.py [hunk 2]
```diff
@@ -1,14 +1,14 @@
-from ..Communication import satprot
+from ..Modules.Satprot import satprot
```
Comment (ron-popov) [inline Tests/Control.py:1]: Can you change the name of the module to something more descriptive ?

**Review by ron-popov — CHANGES_REQUESTED** [review]

File: Tests/Control.py (hunk 2 above)
Comment (shaiTheKimhi) [reply Tests/Control.py:1]: This is yotam's module.
maybe @yotam180 can?

**Comment by yotam180** [comment]
Someone resolve the conflicts with this PR before someone else reviews it again. @ron-popov @shaiTheKimhi @Rogozin-high-school/satellite-contributors

**Comment by ron-popov** [comment]
@shaiTheKimhi Can you resolve the conflict ? All of the code here is yours

**Comment by shaiTheKimhi** [comment]
This branch was started before the master was changed.
Combined with the fact that these changes were not tested with the simulation makes me close this pull request.
@yotam180 should fix the Tester with me so it would work with the simulation as I had a problem connecting to the simulation through satprot.

**Comment by yotam180** [comment]
.........

**Comment by ron-popov** [comment]
@shaiTheKimhi Can i delete this branch ?

**Comment by yotam180** [comment]
Don't ask, just do it.

**Pull request closed without merging**